### DISCUSSION TITLE
Bought a used BCR2000 and have an X-Touch Mini on order

### DISCUSSION THREAD

**Discussion by prohidium** [rsjaffe/MIDI2LR#598] [General]
I decided to give Midi2LR another shot after trying it out several years ago. I used to have a BCF2000 and sold it and regretted that and I just ordered an x-touch mini. While waiting for the new mini, I found a used BCR2000 for sale online and picked it up. The question I have is can I have both units working at the same time if I wanted to keep the mini when it arrives or does anyone recommend the mini over the BCR2000 (from a size/space/workflow perspective) or is the BCR2000 much better. I could cancel the mini and save $100 or so. 

I bought the BCR2000 thinking the extra encoders vs. the mini would give me many more slider controls in LR and almost eliminate any keyboard/mouse interaction when editing. 

Anyone have any input ( pun intended )

**Comment by prohidium** [comment]
UPDATE: The x-touch mini appears to be back ordered and may be for some time. I am proceeding to work with the BCR2000. I have the unit now set up with all of the buttons set to toggle off and I remapped encoders 98 and 99 to 105 and 106. Now to begin to figure out what to assign what!

**Comment by rsjaffe** [comment]
Just be aware that those old Behringer MIDI controllers have buggy drivers. If you have issues, you may have to use a MIDI to USB cable (available for about US$12) instead of a direct controller to computer USB cable. See https://github.com/rsjaffe/MIDI2LR/discussions/565#discussioncomment-1717967

**Reply by prohidium** [reply]
Thanks. I did a panic reset on the unit, set it to USB mode 3, set all buttons to be "toggle off" and then did a store. It is working very well with just a USB cable. Windows 10 correctly identified the BCR2000 as a B-Control. One thing to note is that I had to connect it directly to a USB port on the computer. When I tried it on a USB hub, the unit would load but none of the controls were recognized. Once I removed it from the hub and plugged in directly to a USB port on the PC, it is working properly via USB.

**Reply by rsjaffe** [reply]
Interesting. Maybe I'll give direct connection a try. I've only tried through a hub, due to the way my system is set up.

**Comment by prohidium** [comment]
I have my BCR2000 fully configured (except for 4 buttons) and I have to say that it works very, very well. I almost have too many options to control. With the new masking feature of lightroom, local adjustment sliders have become way more prominent in the workflow. The top 8 rotators are toggled between the main basic and local adjustments and cropping(not used often) and the bottom 3 rows are for editing basic and detail controls for texture, detail and transforming. The buttons are more about those single setting toggles such as contstrain, next, prev, flag, reject and so on. I am not having any issues using this setup! 

![20220122-console1](https://user-images.githubusercontent.com/81877099/150643039-786b8941-3391-4de5-9209-040c7bd20546.JPG)
